src/AppInstallerTestExeInstaller/main.cpp

```diff
@@ -1,15 +1,106 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#include <windows.h>
+#include <winreg.h>
+#include <winerror.h>
 #include <iostream>
 #include <fstream>
 #include <filesystem>
 #include <sstream>
 
-// The installer simply prints all args to an output file
+using namespace std::filesystem;
+
+std::wstring registrySubkey = L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\";

```

Comment:
>wstring [](start = 5, length = 7)

Use std::wstring_view to skip allocation.

Reply:
changed to std::wstring_view